dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataMetadataConnectorTest.java

```diff
@@ -273,9 +275,11 @@ public Void doInConnection(Connection connection)
             // "--database", "db_0"
             );
 
-    MetadataDumper dumper = new MetadataDumper().withExitOnError(false);
+    MetadataDumper dumper =
+        new MetadataDumper(new DumpRunner.Builder().withExitOnError(false).build());
 
     dumper.run(args.toArray(ArrayUtils.EMPTY_STRING_ARRAY));
+    DumpRunner.getInstance().run(new ConnectorArguments());
```

Comment:
The `DumpRunner#run` method is invoked inside the `MetadataDumper#run` method.

Did you mean to execute both `run` methods in here?